src/components/Tooltip/Tooltip.stories.tsx

```diff
@@ -170,7 +170,11 @@ export const UsingIconContent: Story = {
 
 export const UsingImageContent: Story = {
   render: () => (
-    <Tooltip content={<Image src="//placekitten.com/300/300" alt="" />}>
+    <Tooltip
+      content={
+        <Image src="//loremflickr.com/300/300/new+york+public+library" alt="" />
```

Comment:
Nit, you switched these to alphabetical order elsewhere, if you want to do here too.